tests/acceptance/requirements_py3.txt

```diff
@@ -8,8 +8,12 @@ certifi==2023.5.7
     # via requests
 charset-normalizer==2.1.1
     # via requests
+exceptiongroup==1.1.1
+    # via pytest
```

Comment:
This is wrong, actually. It should have been added. I'll update my local pytest and "compile" again this file